### DISCUSSION TITLE
The sensors can be added using the web UI?

### DISCUSSION THREAD

**Discussion by Gelisob** [JaccoR/hass-entso-e#132] [Q&A]
Sorry for the basic question, I do not consider my self a beginner but this quide:

> Configuration:
> The sensors can be added using the web UI.

Just makes me ask, where can I access this web UI?
With normal addons - you can open addon and start-stop and do other things with it, but this is installed through HACS and opening it from HACS just takes you to the github page inside HA. 
I feel kind of stupid, getting stopped so early on. Has this not been a problem for others? 
Maybe we can add a bit more information on the configuration section please? :)

**Comment by sourabhsharma24** [comment]
You can access it here:
Open your Home Assistant dashboard on your computer or phone.
On the left side, click "Configuration."
Next, click "Integrations."
Look for the sensor integration related to what you installed via HACS. It might be labelled as "custom."
Click on it to configure and set up your sensors.

**Comment by Gelisob** [comment]
> You can access it here: Open your Home Assistant dashboard on your computer or phone. On the left side, click "Configuration." Next, click "Integrations." Look for the sensor integration related to what you installed via HACS. It might be labelled as "custom." Click on it to configure and set up your sensors.

Thank you for trying to help but this answer makes it kind of look like you have not really used home assistant before? Or is yours in a different language? 
There is no "configuration" on the menu panel in home assistant. There is "Settings", in there there is no "integrations", there is "devices & services", in there there are tabs for "integrations, devices, entities, helpers". 
But like I said in my original post - things installed through HACS do not appear under addons or integrations in the official settings menu. I can only see it being installed if I go into HACS menu and into it's "integrations" menu. And like I said in the first post - if you click on "ENTSO-e Transparency Platform" there, you will land in the "github" page inside HACS where your options are to just the following (adding picture).
![image](https://github.com/JaccoR/hass-entso-e/assets/22873920/4a068c5e-11c2-4ed3-a956-4251944cb9cd)

Maybe if you have time, you can go and check what are the steps really, that you take to get to the "web UI" that the page is talking about?

**Comment by sourabhsharma24** [comment]
> > You can access it here: Open your Home Assistant dashboard on your computer or phone. On the left side, click "Configuration." Next, click "Integrations." Look for the sensor integration related to what you installed via HACS. It might be labelled as "custom." Click on it to configure and set up your sensors.
> 
> Thank you for trying to help but this answer makes it kind of look like you have not really used home assistant before? Or is yours in a different language? There is no "configuration" on the menu panel in home assistant. There is "Settings", in there there is no "integrations", there is "devices & services", in there there are tabs for "integrations, devices, entities, helpers". But like I said in my original post - things installed through HACS do not appear under addons or integrations in the official settings menu. I can only see it being installed if I go into HACS menu and into it's "integrations" menu. And like I said in the first post - if you click on "ENTSO-e Transparency Platform" there, you will land in the "github" page inside HACS where your options are to just the following (adding picture). ![image](https://user-images.githubusercontent.com/22873920/280546348-4a068c5e-11c2-4ed3-a956-4251944cb9cd.png)
> 
> Maybe if you have time, you can go and check what are the steps really, that you take to get to the "web UI" that the page is talking about?
Hi @Gelisob ,
Thank you for your feedback. I'd like to mention that I was using the GitHub Discussions feature for the very first time, so I wasn't very familiar with how it works. I assumed that the GitHub community is generally supportive, which is why I tried to answer your question even though  I'm not well-versed in that particular concept. but Thanks for providing the process, and I was not aware of it. I'm happy to learn more.

**Reply by Gelisob** [reply]
Sure, maybe it's just not very widely used addon. 
But if you are using it and you have found where the web UI can be found - please let me know. 
I've looked in many ways but this description is either outdated or expects some not very intuitive knowledge about how to get to that configuration web ui.

**Comment by Gelisob** [comment]
still wishing for someone to help with finding this hidden web ui please :)

**Comment by BarryH120** [comment]
I also don't really know how to use home assistant: 
However this worked for me:
1/ added to configuration.yaml:

entsoe:
  api_key: YOUR_API_KEY

2/ restarted HA
3/ got an error message with the option to configure  the entso interface.
4/ there I could enter the API key
5/ deleted the entry under 1/ from the configuration.yaml.

now to get this info in a graph. Got apex charts installed, but what now....

**Reply by Roeland54** [reply]
> I also don't really know how to use home assistant: However this worked for me: 1/ added to configuration.yaml:
> 
> entsoe: api_key: YOUR_API_KEY
> 
> 2/ restarted HA 3/ got an error message with the option to configure the entso interface. 4/ there I could enter the API key 5/ deleted the entry under 1/ from the configuration.yaml.
> 
> now to get this info in a graph. Got apex charts installed, but what now....

I would strongly advise against using the configuration.yaml to setup this integration as home assistant is is moving away from adding integrations through the configuration.yaml as much as possible.
Setup the integration like you would add any other integration to home assistant. 

detailed instructions:
#### Adding integration

1. Go to Settings => Devices and Services
2. Click the button "+ Add integration". 
2. Search for "entso
3. Click on the Entso-e to start te configuration flow.

I updated the readme.md with these instructions.

**Reply by BarryH120** [reply]
> > I also don't really know how to use home assistant: However this worked for me: 1/ added to configuration.yaml:
> > entsoe: api_key: YOUR_API_KEY
> > 2/ restarted HA 3/ got an error message with the option to configure the entso interface. 4/ there I could enter the API key 5/ deleted the entry under 1/ from the configuration.yaml.
> > now to get this info in a graph. Got apex charts installed, but what now....
> 
> I would strongly advise against using the configuration.yaml to setup this integration as home assistant is is moving away from adding integrations through the configuration.yaml as much as possible. Setup the integration like you would add any other integration to home assistant.
> 
> detailed instructions:
> 
> #### Adding integration
> 1. Go to Settings => Devices and Services
> 2. Click the button "+ Add integration".
> 3. Search for "entso
> 4. Click on the Entso-e to start te configuration flow.
> 
> I updated the readme.md with these instructions.

I think something went wrong when I added the integration. Since the configuration flow did not start. Most likely due to all the gaps in my knowledge about HA.
And now I'm struggling with the next step: how to get that graph visible in a dashboard :).
